### ISSUE TITLE
Performance on Activitynet

### ISSUE THREAD

**Issue by srikanth-sfu** [Alvin-Zeng/PGCN#40]
Thanks for releasing your code and features!

I ran the training using the provided default settings on activitynet dataset and the I3D flow features, but obtained mAP much below the reported numbers (10.63 mAP).

**Comment by Jockey721** [comment]
> Thanks for releasing your code and features!
> 
> I ran the training using the provided default settings on activitynet dataset and the I3D flow features, but obtained mAP much below the reported numbers (10.63 mAP).

Hi, Have you successfully reproduced the results now?

**Comment by Jockey721** [comment]
> Thanks for releasing your code and features!
> 
> I ran the training using the provided default settings on activitynet dataset and the I3D flow features, but obtained mAP much below the reported numbers (10.63 mAP).

Did you modify the code in I3D_Pooling.py ?? 
From:
        start_unit = int(min(ft_num-1, np.floor(float(start_ind+off)/interval)))
        end_unit = int(min(ft_num-2, np.ceil(float(end_ind-clip_length)/interval)))
To:
        start_unit = int(min(ft_num-1, np.floor(float(start_ind+off)*100/n_frame)))
        end_unit = int(min(ft_num-2, np.ceil(float(start_ind+off+average_duration)*100/n_frame)))

**Comment by srikanth-sfu** [comment]
Hi,
Thanks. Sorry for the delay in response. -- I didn't change this part. I will check the performance by making this change.

**Comment by Jockey721** [comment]
> Hi,
> Thanks. Sorry for the delay in response. -- I didn't change this part. I will check the performance by making this change.

yeah, i'm running the code of flow feature on ActivitiNet, and the log looks normal now. 
By the way, the RGB feature of ActivityNet provided by authors has some problems, do you have any solutions?
https://github.com/Alvin-Zeng/PGCN/issues/38#issue-691750205

**Comment by srikanth-sfu** [comment]
I haven't tried using RGB features with PGCN.

**Comment by menghuaa** [comment]
> > Thanks for releasing your code and features!
> > I ran the training using the provided default settings on activitynet dataset and the I3D flow features, but obtained mAP much below the reported numbers (10.63 mAP).
> 
> Did you modify the code in I3D_Pooling.py ??
> From:
> start_unit = int(min(ft_num-1, np.floor(float(start_ind+off)/interval)))
> end_unit = int(min(ft_num-2, np.ceil(float(end_ind-clip_length)/interval)))
> To:
> start_unit = int(min(ft_num-1, np.floor(float(start_ind+off)*100/n_frame)))
> end_unit = int(min(ft_num-2, np.ceil(float(start_ind+off+average_duration)*100/n_frame)))

May I ask why this change was made?
